feedback/static/html/feedback_instructor.html

```diff
@@ -24,7 +24,7 @@
           {% for block in blocks %}
           <tr role="row" class="pgn__data-table-row">
             <td colspan="6" role="cell" class="pgn__data-table-cell-wrap">
-              <a class="admin-panel" href="{{ block.url }}"><button class="btn btn-primary" ="button">{% trans "See Unit" %}: {{ block.parent }}</button></a>
+              <a class="admin-panel" href="{{ block.url }}"><button class="btn btn-primary" ="button">{{ block.section_display_name }} > {{ block.subsection_display_name }} > {{ block.unit_display_name }} > {{ block.display_name }}</button></a>
```

Comment:
I'm curious about how this will appear in RTL languages. Could you please test it and share a screenshot of how it looks?

Reply:
Hi @farhan, thanks for the review! Sure, that's how it looks like:

![image](https://github.com/openedx/FeedbackXBlock/assets/64033729/d6409e24-c96c-4d33-9d8c-ee4cba9b6d31)